src/main/java/Models/TrainingProgram.java

```diff
@@ -0,0 +1,68 @@
+package Models;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class TrainingProgram  implements Training {
```

Comment:
this is the only class that implements this interface - maybe we don't need the interface at this moment?